supervisor/monitor_solaris.go

```diff
@@ -0,0 +1,38 @@
+package supervisor
+
+import (
+	"errors"
+
+	"github.com/docker/containerd/runtime"
+)
+
+func NewMonitor() (*Monitor, error) {
+	return &Monitor{}, nil
+}
+
+type Monitor struct {
+	ooms chan string
+}
+
+func (m *Monitor) Exits() chan runtime.Process {
```

Comment:
This was meant as just a build clean PR..to get containerd building on Solaris. 
Once we have CI for Solaris in docker we will submit a PR for a functionally complete containerd.